js/src/logger.ts

```diff
@@ -274,6 +274,27 @@ class BraintrustState {
     this._logConn = null;
   }
 
+  public copyLoginInfo(other: BraintrustState) {
+    this.appUrl = other.appUrl;
```

Comment:
How about `appPublicUrl`?

Reply:
good catch. also cleared in the other case.